scopes/workspace/workspace/workspace.ts

```diff
@@ -655,7 +655,13 @@ export class Workspace implements ComponentFactory {
         this.logger.warn(`unable to get lane's data from a remote due to an error:\n${err.message}`);
         return null;
       }
-      throw err;
+      /** *
+       * Note: Currently, when it is a valid scope and the lane is not found,
+       * it doesn't throw a LaneNotFound error.
+       * Instead, it throws a General Error
+       */
+      // throw err;
+      return null;
```

Comment:
It would be best to check for a specific error, because otherwise, some random errors (e.g. some temp syntax errors in the code) will cause this to return null unexpectedly. 
anyway, it seems like the LaneNotFound error is fixed in this PR by giving it a code number, so why not checking whether it's a LaneNotFound error?

Reply:
Yes, that is my intention. To not return null and throw the error but I couldn't figure out a way to test this. So left this until I can test.

Reply:
got it. makes sense.